app/assets/stylesheets/modules/panel_list.css.scss

```diff
@@ -1,4 +1,4 @@
-$panel-list-item-height: 85px;
+$panel-list-item-height: 75px;
```

Comment:
We're requesting a 65px image from the browse scaler, but it ends up being drawn at 75px on the screen. Please update the browse scaler URL to request the correct size.